backend/src/test/java/com/google/sps/TimeTest.java

```diff
@@ -0,0 +1,51 @@
+package com.google.sps;
+
+import com.google.sps.data.Time;
+import java.util.Date;
+import java.util.Calendar;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** */
+public final class TimeTest {
+
+  private Time time;
+
+  @Before
+  public void setUp() {
+    // Set "current" date to  1/1/2020 2:00pm GMT
+    Calendar c = Calendar.getInstance();
+    c.set(2020, 0, 1, 14, 0, 0);
```

Comment:
What do you think about using a date formatter to try to establish a common way to set dates and potentially catch any date errors? This is the documentation for the [SimpleDateFormat](https://docs.oracle.com/javase/8/docs/api/java/text/SimpleDateFormat.html) and I also found an [example](https://mkyong.com/java/java-convert-date-and-time-between-timezone/) of how it's used with the Calendar and other classes since I know in the future we might need to convert between timezones.

Reply:
+1 to encourage using standard libraries whenever possible

Reply:
Right now for the prototype, everything is in the long timestamp format, but for the MVP and depending on user input format, I'll make it a more standardized time format.

Reply:
Ok, sounds good